pmm-app/src/pmm-qan/panel/qan.constants.ts

```diff
@@ -443,7 +440,7 @@ export const METRIC_CATALOGUE = {
   },
   innodb_io_r_wait: {
     humanizeName: 'Innodb IO Read Wait',
-    tooltipText: 'Shows how long (in seconds) it took InnoDB to actually read the data from storage',
+    tooltipText: 'Time (in seconds) for InnoDB to read the data from storage',
```

Comment:
@lunaticusgreen , this is about Units again. can you check and confirm - that we can drop all units names from descriptions?

Reply:
Unless units are universal and known, better to leave them in. Alternative phrasing could be "Number of seconds for ..."